### PR TITLE
[Easy] Airlock Integration

### PR DESCRIPTION
**Pull request by dfangshuo**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # "Describe what's new in this PR"
Integrate Airlock, without Access Resolvers (with the exception of 1) (rest will be included in the next PR).

@ethanlee16 should be a quick review, basically what we talked about

### How to Review

[//]: # 'The order in which to review files and what to expect when testing locally'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc"

### Next steps

[//]: # "What doesn't work yet, what's NOT in this PR/has to be done "

### Tests Performed, Edge Cases

[//]: # 'If you made changes/additions to `airtable.js` make sure you update/add'
[//]: # '[at least 1] corresponding unit tests in `airtable.spec.js`'

### Screenshots

[//]: # "Add screenshots!!! If you'd like"

CC: @dfangshuo

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,9 +22,19 @@ dotenv.config(); // Set up environment variables
 const app = express();
 const port = process.env.PORT || 3000;
 
+/* eslint-disable no-new */
+new Airlock({
+  server: app,
+  airtableApiKey: [apiKey],
+  airtableBaseId: BASE_ID,
```
Comment (aivantg) [inline index.js:29]: I would recommend getting the API key and BASE ID directly from `process.env`, seems a little but circuitous to import it from airtable? but I understand the intention so I don't mind either way

File: resolvers/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  'Project Group': {
+    read: (projectGroup, authedUser) => {
```
Comment (aivantg) [inline resolvers/index.js:3]: presumably these will be updated? LOL

File: index.js [hunk 3]
```diff
@@ -19,9 +22,19 @@ dotenv.config(); // Set up environment variables
 const app = express();
 const port = process.env.PORT || 3000;
 
+/* eslint-disable no-new */
```
Comment (aivantg) [inline index.js:25]: could we add some comments here mentioning what Airlock is? The rest of the functions are loosley explained with the `console.log` statements, might be worth making a note what this is

File: index.js [hunk 4]
```diff
@@ -19,9 +22,19 @@ dotenv.config(); // Set up environment variables
...
+  airtableBaseId: BASE_ID,
+  airtableUserTableName: 'Owner',
+  airtableUsernameColumn: 'Email',
+  airtablePasswordColumn: 'Password'
+});
+
 app.use(cors());
 app.use(express.json());
-app.use(express.static('temp')); // Make PDFs accessible
+app.use(express.static(Constants.TEMP_BILL_SAVE_FOLDER_NAME)); // Make PDFs accessible
```
[the remark this answers is not in the document]
Comment (dfangshuo) [reply index.js:37]: hmm so i believe ive already went ahead and made that change. The main motivation for doing that was so all of the temp folders are tightly coupled to 1 variable, which would make for 1) ease of change 2) prevention of weird bugs where you accidentally change the name 1 place and don't the other

File: resolvers/index.js (hunk 2 above)
Comment (dfangshuo) [reply resolvers/index.js:3]: yes! this is just 1 set of access resolvers haha. ideally this shouldn't be in this PR at all, but...
